pkg/rbd/driver.go

```diff
@@ -133,8 +133,16 @@ func (r *Driver) Run(conf *util.Config, cachePersister util.CachePersister) {
 	// Create GRPC servers
 	r.ids = NewIdentityServer(r.cd)
 
+	topology := map[string]string{}
 	if conf.IsNodeServer {
-		r.ns, err = NewNodeServer(r.cd, conf.Vtype)
+		topology, err = util.GetTopologyFromDomainLabels(conf.DomainLabels, conf.NodeID, conf.DriverName)
+		if err != nil {
+			klog.Fatalln(err)
+		}
+	}
+
+	if conf.IsNodeServer {
```

Comment:
we can remove this change and move below code to line 142

Reply:
Addressed in the latest update.